package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Might be worth moving this + the package-lock to the same folder as the tests, means you can open them and run them from within the same folder in an editor without needing to open the whole project with the .NET files alongside it

Reply:
This would then mean you dont need the changes in the `cypress.json` to point to all the tests too